source/codegen/metadata/niscope/config.py

```diff
@@ -1,12 +1,12 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-SCOPE API metadata version 23.0.0f168
+# This file is generated from NI-SCOPE API metadata version 23.3.0d9999
```

Comment:
await: This is coming from the output of the local changes in this [AzDo PR](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/434965). Once that goes in I'll update this to come from an actual export.

Reply:
Updated to pull from 23.3.0d127